org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -0,0 +1,81 @@
+define([
+  'common/modules/services',
+], function (module) {
+  'use strict';
+
+  module.factory('FileUploadService', ['$q', 'FileUploader',
+    function ($q, FileUploader) {
+      var fileUploader;
```

Comment:
Why do you have this variable here? That means that you're gonna override any file uploader that was created with the newly created one, since you're storing each file uploader in the same variable